bucket/sioyek.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "1.1.0",
+    "description": "A PDF viewer designed for reading research papers and technical books",
+    "homepage": "https://sioyek.info/",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ahrm/sioyek/releases/download/v1.1.0/sioyek-release-windows-portable.zip",
+            "hash": "29c65d4a9932199f4cdf46aca5950518d94dec27aca03e6ebaeee88121bea8f4"
+        }
+    },
+    "extract_dir": "sioyek-release-windows",
+    "pre_install": "New-Item \"$dir\\last_document_path.txt\" | Out-Null",
```

```suggestion
    "pre_install": [
        "if (!(Test-Path \"$persist_dir\\last_document_path.txt\"))New-Item \"$dir\\last_document_path.txt\" | Out-Null",
        "'test.db', 'local.db', 'shared.db' | ForEach-Object {",
        "    if (Test-Path \"$persist_dir\\$_\") {Copy-Item \"$persist_dir\\$_\" \"$dir\\\"}",
        "}"
    ],
```

Those `.db` files will be created after running the app. Also an empty `.db` file will cause the app to crash, so we need to persist them **manually**.

Related code can be found [here](https://github.com/ahrm/sioyek/blob/cffcdcb8381a432fd0c81c99d4499a50ba3324d2/pdf_viewer/main.cpp#L210)